ci-operator/config/openshift/cluster-policy-controller/openshift-cluster-policy-controller-release-4.5.yaml

```diff
@@ -17,9 +17,6 @@ images:
 - dockerfile_path: Dockerfile.rhel
   from: base
   inputs:
-    bin:
-      as:
-      - registry.svc.ci.openshift.org/ocp/builder:golang-1.12
```

Comment:
@soltysh do you have a followup to https://github.com/openshift/cluster-policy-controller/commit/036d17e0e9d5335c271e1d4303b589e828d5a351 ?

Reply:
Yes I had in https://github.com/openshift/release/pull/9833